PyYapf.py

```diff
@@ -430,20 +428,12 @@ def on_pre_save(self, view):  # pylint: disable=no-self-use
 
 
 def get_setting(view, key, default_value=None):
-    settings = sublime.load_settings(SETTINGS_FILE)
-    value = settings.get(key, default_value)
-    # check for project-level overrides:
-    project_value = _get_project_setting(key)
-    if project_value is None:
-        return value
-    return project_value
-
-
-def _get_project_setting(key):
+    # 1. check sublime settings (this includes project settings)
     settings = sublime.active_window().active_view().settings()
```

Comment:
Can this ever return None? (ie, run from a non-project environment?) If so, your changes may break.

Reply:
My understanding is that `settings()` returns the views settings, which are obtained from the user's `Preferences.sublime-settings` and refined by the current project's settings if run in a project environment.

Reply:
That is, thanks to your PR one can now also configure PyYapf by modifying the global `Preferences.sublime-settings` file!

Reply:
Ok, sounds good to me!